Makefile

```diff
@@ -11,7 +11,8 @@ NODE ?= node
 ZUUL ?= $(NODE) $(BIN)/zuul
 
 test:
-	@if [ "x$(BROWSER_PLATFORM)" = "x" ]; then \
+	@$(BIN)/mocha test/*.js; \
+	if [ "x$(BROWSER_PLATFORM)" = "x" ]; then \
```

Comment:
It should be set up something like this: https://github.com/TooTallNate/plist.js/blob/0cf3035bb86b074e6cc06df3857f51761a6cf70a/Makefile#L48-L74